rest-api/tools/import_organizations.py

```diff
@@ -152,6 +153,7 @@ def _entity_from_row(self, row):
     except TypeError:
       logging.warn('Invalid site status %s for site %s', row[SITE_STATUS_COLUMN], google_group)
       return None
+    enrolling_status = EnrollingStatus(row[ENROLLING_STATUS_COLUMN].upper())
```

Comment:
If you're going to require this column to be present in the data, you should add it to the array pf required fields passed into the super constructor on line 117.

However we need to make sure that field actually gets added before we can do this.

You may also want to catch TypeError here the way we did for SiteStatus (in case we get some value other than ACTIVE or INACTIVE.)

Reply:
done